pandas/compat/_optional.py

```diff
@@ -59,7 +66,8 @@ def get_version(module: types.ModuleType) -> str:
 
     if version is None:
         raise ImportError(f"Can't determine version for {module.__name__}")
-    return version
+    # psycopg2 appends " (dt dec pq3 ext lo64)" to it's version
+    return version.split()[0]
```

Comment:
Can you special case this for psycopg2 only?

Reply:
Sure, changed.